payments/management/commands/delete_discount_code.py

```diff
@@ -0,0 +1,61 @@
+from datetime import datetime
+
+import pytz
+from django.core.management import BaseCommand
+
+from payments.models import BulkDiscountCollection, Discount
+from unified_ecommerce import settings
+
+
+class Command(BaseCommand):
+    """
+    Deactivates multiple discounts using the Discount codes.
+    An example usage of this command:
+    python manage.py delete_discount_code --discount_codes 1 2 3
+    """
+
+    help = "Deactivate multiple discounts using the Discount codes."
+
+    def add_arguments(self, parser) -> None:
+        """
+        Add arguments to the command parser.
+        """
+        parser.add_argument(
+            "--iscount_codes",
+            type=int,
```

Comment:
nit: `discount-codes`, not`iscount_codes`
also: needs to be `type=str` since it takes a code now